homeassistant/components/withings/common.py

```diff
@@ -0,0 +1,184 @@
+"""Common code for Withings."""
+import time
+import datetime
+import logging
+import re
+
+import nokia
+from oauthlib.oauth2.rfc6749.errors import MissingTokenError
+from requests_oauthlib import TokenUpdated
+
+from homeassistant.exceptions import PlatformNotReady
+from homeassistant.util import slugify, Throttle
+from . import const
+
+_LOGGER = logging.getLogger(const.LOG_NAMESPACE)
+NOT_AUTHENTICATED_ERROR = re.compile(".*(Error Code (100|101|102|200|401)|Missing access token parameter).*", re.IGNORECASE)  # pylint: disable=line-too-long  # noqa: E501
```

Comment:
Please break the line instead of disabling too long line. Break the line after the first parenthesis.